### DISCUSSION TITLE
Tasmota Web GUI no longer accessible after upgrade

### DISCUSSION THREAD

**Discussion by tappas197112** [arendst/Tasmota#21356] [Support request]
Hi,

yesterday I updated my 12 Gosund sockets with Tasmota (ESP 8266) from version 12.x to the current 13.4.1. Always via the WebUI with the current OTA URL to the German binary.

However, the last socket did not want to be updated via WebUI or with direct upload. In this respect, I first went to the minimal version, then to the full version 13.4.1.

After rebooting I can no longer access the web GUI, nothing comes back. However, you can issue commands normally via browser as well as via mqtt (Iobroker with Sonoff adapter). Maybe a little slow at times but it works.

I've tried everything to go back to a previous version, but it doesn't work. Out of desperation, I tried Reset 6, but it didn't help either. After that i tried Reset 1 and connected back to my network. Does'nt help, Web UI is not accessible. Attached is the status output.

Do you have any ideas?

`{"Status":{"Module":1,"DeviceName":"Tasmota","FriendlyName":["Tasmota"],"Topic":"tasmota_BB24A7","ButtonTopic":"0","Power":"0","PowerLock":"0","PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0},"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-DE.bin.gz","RestartReason":"Exception","Uptime":"0T00:06:36","StartupUTC":"","Sleep":50,"CfgHolder":4617,"BootCount":18,"BCResetTime":"1970-01-01T00:00:00","SaveCount":22,"SaveAddress":"F6000"},"StatusFWR":{"Version":"13.4.1.2(ca5a2d3-tasmota)","BuildDateTime":"2024.05.04 18:05:58","Boot":31,"Core":"2_7_6","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8285N08","CR":"370/699"},"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["CIA",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A192800000000","00000080","00006000","00004000","00000000"]},"StatusMEM":{"ProgramSize":648,"Free":352,"Heap":22,"ProgramFlashSize":1024,"FlashSize":1024,"FlashChipId":"144051","FlashFrequency":40,"FlashMode":"DOUT","Features":["0407","8F9AC787","04368001","000000CF","010013C0","C000F981","00004004","00001000","54000020","00000080","00000000"],"Drivers":"1,2,!3,!4,!5,!6,7,!8,9,10,12,!16,!18,!19,!20,!21,!22,!24,26,!27,29,!30,!35,!37,!45,62,!68","Sensors":"1,2,3,4,5,6","I2CDriver":"7"},"StatusNET":{"Hostname":"tasmota-BB24A7-1191","IPAddress":"10.10.10.140","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.1.1","DNSServer2":"0.0.0.0","Mac":"D8:F1:5B:BB:24:A7","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0},"StatusMQT":{"MqttHost":"","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_BB24A7","MqttUser":"DVES_USER","MqttCount":0,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4},"StatusTIM":{"UTC":"1970-01-01T00:06:34Z","Local":"1970-01-01T00:06:34","StartDST":"1970-01-01T00:00:00","EndDST":"1970-01-01T00:00:00","Timezone":"+00:00","Sunrise":"20:13","Sunset":"05:47"},"StatusSNS":{"Time":"1970-01-01T00:06:34"},"StatusSTS":{"Time":"1970-01-01T00:06:34","Uptime":"0T00:06:36","UptimeSec":396,"Heap":22,"SleepMode":"Dynamic","Sleep":0,"LoadAvg":119,"MqttCount":0,"POWER":"OFF"},"StatusSTK":{"Exception":28,"Reason":"Exception","EPC":["40258c32","00000000","00000000"],"EXCVADDR":"0047c264","DEPC":"00000000","CallChain":["402592f4","40259de5","402593be","402592f4","402597f3","402593be","40258b5d","40258b5d","4024695c","40101c5b","401019d3","40258ba0","40291824","40101c92","40258c24","402586c4","4024eee5","40291824","40101c5b","40251c6c","401019d3","4024efb0","402299f3","4025191c","40229bb0","402095b2","4028c825","4022dd3e","401019d3","40101c5b","402966bc"]}}`

**Comment by Jason2866** [comment]
There was a bug in the development firmware yesterday. Your used version is from this time `ca5a2d3`.
The bug is fixed with commit `c4e847d2` from today.
Maybe it works to initiate a update via mqtt command

**Comment by tappas197112** [comment]
Thanks, Jason2866

Unfortunately I don't have access via mqtt since I tried "Reset 1". Via browser url it doesn't seem to work. I tried:

http://10.10.10.140/cm?cmnd=OtaUrl%20url%20http://ota.tasmota.com/tasmota/tasmota-DE.bin.gz
Output: {"OtaUrl":"url http://ota.tasmota.com/tasmota/tasmota-DE.bin.gz"}

then http://10.10.10.140/cm?cmnd=Upgrade%201
Output: {"Upgrade":"Version 13.4.1.2 from url http://ota.tasmota.com/tasmota/tasmota-DE.bin.gz"}

Without any response after 5 minutes...

**Reply by Jason2866** [reply]
You wrote the device is problematic to upgrade before the issue with the Web broken firmware happened.
Anyways it updated to the version from yesterday. When not calling the devices Web Frontend the firmware works so far. So it should be possible to do an upgrade. You might move the device to a different place, maybe wifi is not optimal where it is currently.

**Comment by tappas197112** [comment]
Thanks for your help. After a while and a lot of tries, the upgrade was possible via "Upgrade 1". After that, I downgraded to 13.4.0 and everything works fine again. Thank you very much!

**Reply by Jason2866** [reply]
Glad to hear no hardware deassembly work is needed!
